etc/pip-requires-test

```diff
@@ -6,7 +6,7 @@ pytest-xdist~=2.2.0; python_version >= '3.6'      # Used for parallel testing on
 pyflakes==2.1.1                                   # Passive checker of Python programs
 flake8==3.7.8                                     # Wrapper around PyFlakes&pep8
```

Comment:
update to 3.9.2 ?

Reply:
see https://github.com/rucio/rucio/pull/4588